websites/docs-smoke-test/src/release-notes/bugfix.mdx

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Wondering if it makes sense to have a "draft" flag or something. Probably not, just double checking that.

Reply:
Jekyll has  both time-based and explicit via "draft" frontmatter logic and we went lengths to deactivate both in the current site because it interferes in ugly ways with a git branch/merge based publishing workflow.  
So for the `commercetools-docs` It's a feature to not have a draft flag. Unpublished drafts are a git branch together with the matching documentation changes. 

This may be different in repos like the application kit - there branching is decoupled from documentation changes at least to some degree and I can imagine you want to avoid long-open branches there?

Reply:
Yeah I thought so. Thanks

Reply:
I'll ad an optional "published" key to the spec, but not the template now which can be used for both preventing publication if you have to merge to master for some reason and also to be able to hide old release notes that have to be removed for some reason.   "draft" would not be intuitive for the latter case.